test/views/pages/confirmation/confirmFPSpec.scala

```diff
@@ -0,0 +1,178 @@
+        }
+      }
+
+      "have a H1 tag that" should {
+
+        lazy val h1Tag = doc.select("H1")
+
+        s"have the page heading '$plaConfirmFP16PageHeading'" in {
+          h1Tag.text shouldBe {
+            plaConfirmFP16PageHeading
+          }
+        }
+
+        "have the heading-large class" in {
+          h1Tag.hasClass("heading-large") shouldBe true
+        }
+      }
+
+      s"have the first declaration paragraph of $plaConfirmFP16Para1" in {
+        doc.body.select("p").get(0).text shouldBe plaConfirmFP16Para1
+      }
+
+      s"have the second declaration paragraph of $plaConfirmFP16Para2" in {
+        doc.body.select("p").get(1).text shouldBe plaConfirmFP16Para2
+      }
+
+      "has a list of bullet points that" should {
+
+        s"have the first bullet of $plaConfirmFP16Bullet1" in {
+          doc.body.select("ul li").get(0).text shouldBe plaConfirmFP16Bullet1
+        }
+
+        s"have the second bullet of $plaConfirmFP16Bullet2" in {
+          doc.body.select("ul li").get(1).text shouldBe plaConfirmFP16Bullet2
+        }
+
+        s"have the third bullet of $plaConfirmFP16Bullet3" in {
+          doc.body.select("ul li").get(2).text shouldBe plaConfirmFP16Bullet3
+        }
+      }
+
+      s"have the first declaration paragraph of $plaConfirmFP16Para3" in {
+        doc.body.select("p").get(2).text shouldBe plaConfirmFP16Para3
+      }
+
+      "have help text that" should {
+
+        lazy val helpText = doc.body.select("summary").text
+        lazy val helpLink = doc.body.select("details summary")
+
+        s"have the label text $plaConfirmFP16Help" in {
+          helpText shouldBe plaConfirmFP16Help
+        }
+
+        "have the class 'visuallyhidden'" in {
+          helpLink.select("span").size shouldBe 1
+        }
+      }
+
+      "has help paragraphs which" should {
+
+        lazy val accordionText = doc.body.select("details div p")
+
+        s"have the text $plaConfirmFP16HiddenPara1" in {
+          accordionText.get(1).text shouldBe plaConfirmFP16HiddenPara1
+        }
+
+        s"have the text $plaConfirmFP16HiddenPara2" in {
+          accordionText.get(2).text shouldBe s"$plaConfirmFP16HiddenPara2 $plaConfirmFP16HiddenParaLinkText ($plaBaseNewWindow)"
+        }
+
+        s"have the class 'visuallyhidden'" in {
+          doc.body.select("details div").size shouldBe 1
+        }
+      }
+
+      "have an external help link within the help paragraphs which" should {
+
+        lazy val accordionLink = doc.select("a#fp16-help-link")
+
+        s"have the link text $plaConfirmFP16HiddenParaLinkText ($plaBaseNewWindow)" in {
+          doc.body.select("details div p a").get(0).text shouldBe s"$plaConfirmFP16HiddenParaLinkText ($plaBaseNewWindow)"
+        }
+
+        "have the correct link destination" in {
+          accordionLink.attr("href") shouldBe "https://www.gov.uk/hmrc-internal-manuals/pensions-tax-manual/ptm093100"
+        }
+
+        "have the correct link ID" in {
```

Comment:
ID tests are not useful as the ID here is not a required component (dynamic or not)